app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ShoppingListsController < ApplicationController
+  before_action :find_user
+
+  def index
+    @total_foods = 0
```

Comment:
- [ ] Hi great developers! I so much appreciate your tenacity up to this level, I oberved that instead of the app to add the total number of food itmes; it actually concatenates it. 👍 Kindly adjust it. ℹ️ 

<img width="642" alt="Untitled" src="https://github.com/Itswali/Recipe-App/assets/103386183/1133d9f0-ed44-435b-8374-b4fdde7886f4">

Reply:
Its As Per Design In quantity show number of Items to use in recipe and its measurement unit.

Reply:
Thank you @OyePriscilla for your comment.I am reaching out to you to say that we have designed the shopping list as per the wire frame design and as per the project requirements.Kindly refer to the project requirements for more clarification